tensorflow/core/kernels/dynamic_partition_op_gpu.cu.cc

```diff
@@ -337,9 +337,10 @@ class DynamicPartitionOpGPU : public AsyncOpKernel {
     // Determine temporary device storage requirements.
     Tensor cub_temp_storage;
     size_t temp_storage_bytes = 0;
-    gpuprim::DeviceRadixSort::SortPairs(
-        NULL, temp_storage_bytes, partitions_ptr, partitions_out_ptr,
-        indices_in_ptr, indices_out_ptr, N, 0, sizeof(int32) * 8, cu_stream);
+    CHECK_EQ(gpuprim::DeviceRadixSort::SortPairs(
```

Comment:
Can you please change the CHECK_EQ calls in this file in OP_REQUIRES_OK_ASYNC if possible?
We try to avoid CHECK macros in new code.

Reply:
Thanks for the feedback. 

I am not 100% sure what works best for the new code, wondering do you have any recommendation on it? Does TF_CHECK_OK meet the requirement?

Reply:
Thanks will do it.

Reply:
@akuegel Thanks for the suggestion. I have replaced CHECK_EQ with OP_REQUIRES_ASYNC, which is also used in this source file. OP_REQUIRES_OK_ASYNC will cause ERROR in the build as the status is either cudaSuccess or hipSuccess instead of Okstatus();

Please take another look, thanks.